app/src/main/java/com/github/se/polyfit/ui/components/NutritionalInformation.kt

```diff
@@ -48,16 +65,55 @@ fun NutritionalInformation(mealViewModel: MealViewModel) {
   }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
-private fun MealName(mealName: String) {
+private fun MealName(mealName: String = "", onNameChange: (String) -> Unit = {}) {
```

Comment:
Likely don't need to include defaults for this
```suggestion
private fun MealName(mealName: String, onNameChange: (String) -> Unit) {
```

Reply:
done